### ISSUE TITLE
🐛 Apply donor-level exceptions on given entities give `400` errors

### ISSUE THREAD

**Issue by lindaxiang** [icgc-argo/roadmap#1184]
## Describe the bug

We’ve implemented program-level and donor-level exceptions:
- program-level exceptions allow the exceptions on requested fields for all the donors in the program
- donor-level exceptions allow the exceptions on requested fields of given entity for requested donors. 

We should support the programs to request both program and donor level exceptions as long as they are applied on different fields. 
However, when I tried to apply donor-level exceptions, it gives me `400` error with message: 
`Program exception already exists`.

## Steps To Reproduce

Steps to reproduce the behaviour:
1. Go to 'https://clinical.argo-qa.cumulus.genomeinformatics.org/api-docs/#/Exception/post_exception_property__programId__entity'
2. Provide your access JWT token
3. Click on 'Try it out'
5. Input `MONSTAR-JP` as programId
6. Click on `Choose File` by submitting the file
[Monstar-jp-donorExceptionFile_specimen_Oct08.txt](https://github.com/user-attachments/files/17295397/Monstar-jp-donorExceptionFile_specimen_Oct08.txt)
7. See the error: `400` with message: `Program exception already exists`.

**Note**: You can check the Program-level exceptions for MONSTAR-JP, and it does not have exceptions on the requested fields. 

## Expected behaviour
It should allow me to apply both both program and entity level exceptions at the same time as long as they are applied on different fields.

**Comment by lindaxiang** [comment]
Program level exception file for testing in QA:
[Monstar-jp-programExceptionFile_Sep11-qa.txt](https://github.com/user-attachments/files/17318315/Monstar-jp-programExceptionFile_Sep11-qa.txt)

**Comment by lindaxiang** [comment]
Test environment: QA
Test program: MONSTAR-JP

I am able to apply the donor-level exceptions for entity of `Specimen` with the following test file: 
[Monstar-jp-donorExceptionFile_specimen_Oct08.txt](https://github.com/user-attachments/files/17295397/Monstar-jp-donorExceptionFile_specimen_Oct08.txt)
The clinical data submissions also work as expected. It will allow me to submit the records with requested exception values. 

However, when I continued to apply the donor-level exceptions for another entity of `Treatment` with the following file:
[Monstar-jp-donorExceptionFile_treatment_Oct08.txt](https://github.com/user-attachments/files/17345885/Monstar-jp-donorExceptionFile_treatment_Oct08.txt)
It gave me the following `500` error:
```
{
  "error": "ValidationError",
  "message": ""
}
```
I can't see the details of the error, so not sure what's going wrong.

**Comment by joneubank** [comment]
Super simple fix here, in the treatment file the `requested_exception_value` column is case sensitive, has to have the value `Unknown` or `Not applicable`.

**Comment by lindaxiang** [comment]
Yes, the applying of donor-level exceptions on given entities work properly after correct the `requested_exception_value` as what Jon has pointed out.
